.template/addons/heroku/config_template.rb

```diff
@@ -0,0 +1,11 @@
+insert_into_file 'config/environments/production.rb', before: /%r{end\nend\Z}/ do
+  <<~CODE
+    config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
```

Comment:
I think we don't need this line. As we had [this](https://github.com/nimblehq/rails-templates/pull/242)

Reply:
@gutakk Thanks for the good catch, resolved https://github.com/nimblehq/rails-templates/pull/238/commits/fb49d58544d911d81c69b8da4041687dfa85a460 🤝